src/validators/v1/organization.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	update: (req) => {
+		req.checkBody('mentee_feedback_question_set').notEmpty().withMessage('mentee_feedback_question_set is empty')
```

Comment:
@sumanvpacewisdom It should be optional, if either one is passed, both are required 

otherwise how do you update the other fields  ?